src/app/view/components/_common/dot-page-selector/dot-page-selector.component.html

```diff
@@ -32,3 +32,9 @@
         [suggestions]="results.data">
     </p-autoComplete>
 </ng-template>
+
+<dot-icon-button class="dot-page-selector__hint" icon="help_outline" (click)="hint.toggle($event)"></dot-icon-button>
```

Comment:
This should be a component like `field-helper` that receive the html or string to display.

Reply:
And reuse in the other field.